### ISSUE TITLE
Text in an image 1.4.10

### ISSUE THREAD

**Issue by WayneEDick** [w3c/wcag#904]

**Comment by WayneEDick** [comment]
The solutions for text in an image, create equal alt text, does not solve the reflow problem. 

The waiver for images should not apply to text in an image.

**Comment by JAWS-test** [comment]
Maybe one should distinguish between the following text graphics
* a word or group of words (e.g. a button or heading): violation of 1.4.10, if scrolling is necessary
* a continuous text: violation of 1.4.10 if scrolling is required
* a complex graphic that also contains text (e.g. https://upload.wikimedia.org/wikipedia/commons/thumb/1/14/LOM_base_schema.png/1920px-LOM_base_schema.png): no violation of 1.4.10.

However, the first two forms are also violations of SCs 1.4.5 and 1.4.9 and the complex graphic requires a complete text alternative according to SC 1.1.1, which in turn should satisfy 1.4.10.

**Comment by WayneEDick** [comment]
Thank You,
That is a very good breakdown.
Of course, one can usually find the text version of a database diagram in
the data dictionary. Still, the primary issue is continuing text in images.
Best, Wayne

**Comment by JAWS-test** [comment]
It would certainly make sense to include a note in the Understanding that the exception "images" does not apply to "images of text" (at least for continuous text and individual words or phrases)

**Comment by alastc** [comment]
I think this needs teasing out a bit more.

> The waiver for images should not apply to text in an image.

The 'waiver' in reflow is "Except for parts of the content which require two-dimensional layout for usage or meaning". It doesn't normatively say "images", the bar is whether 2D layout is need to preserve meaning.

 At the one end of the scale, dropping a paragraph of standard text into an image is a clear fail of 1.4.5 images of text, and could well trigger a fail of text-sizing and reflow.

In the database schema example above we'd also have to consider 1.4.5, 1.1.1 text alternatives, and others, it is a complex example.

I'm not really sure what the trigger for the comment was, so it's hard to suggest a concrete outcome.

**Comment by JAWS-test** [comment]
> The 'waiver' in reflow is "Except for parts of the content which require two-dimensional layout for usage or meaning". It doesn't normatively say "images", the bar is whether 2D layout is need to preserve meaning.

And the note 1 says:

> Examples of content which require two-dimensional layout **are images**, maps, diagrams, video, games, presentations, data tables, and interfaces where it is necessary to keep toolbars in view while manipulating content.

It would be better here
* either a formulation of the type: "Examples of content which require two-dimensional layout **could be** images, maps ..." so that it becomes clear that not every graphic automatically fulfills 1.4.10.
* or, if the note cannot be changed (if it is normative), an addition in the Understanding Text

**Comment by alastc** [comment]
Ok, but given the existance of 1.4.5 where text in images would already fail, what is the scenario where something passes that should not?

I get that alt text doesn't help, but that isn't a [technique for 1.4.5](https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html#techniques).

**Comment by JAWS-test** [comment]
> 1.4.5 Customizable:  The image of text can be visually customized (the font, size, color, and background can be set) to the user's requirements

An SVG graphic can fulfill SC 1.4.5 because it can be visually customized and still violates 1.4.10 because reflow is not possible. I am not saying that every SVG meets 1.4.5, but it is possible to design SVG to meet the exception of 1.4.5.

**Comment by alastc** [comment]
And SVGs can have size-based media queries applied as well, so they can fulfil reflow as well. (It's very rare in practice, but I've seen demos from Doug Schepers.)

**Comment by JAWS-test** [comment]
> And SVGs can have size-based media queries applied as well, so they can fulfil reflow as well.

YES, but reflow is not required in SC 1.4.5 and SC 1.4.10 only says that images are an exception. Exactly for this reason a hint should be inserted in 1.4.10 that the exception does not apply to all images in general.
